packages/@aws-cdk/aws-stepfunctions-tasks/test/integ.run-batch-job.ts

```diff
@@ -0,0 +1,63 @@
+import * as batch from '@aws-cdk/aws-batch';
+import * as ecs from '@aws-cdk/aws-ecs';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import * as path from 'path';
+import * as tasks from '../lib';
+
+class RunBatchStack extends cdk.Stack {
```

Comment:
Please add verification steps to this test as indicated in the [contribution guide](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#integration-tests).